### PR TITLE
Contribute load testing and mock api

### PR DESCRIPTION
**Pull request by EliiseS**
This is a PR to contribute back the load testing framework built by our dev crew (@stuartleeks @lawrencegripper @storey247 @martinpeck @damoodamoo @bart-jansen) at Microsoft.

PR contains:
- Locust load testing framework + load test scenarios for the Databricks Operator
- Databrick Mock API used to run the load tests against
- Install python and initalize virutal enviroment in dev container
- Load testing and MockAPI dashboards in grafana
- Make commands to:
   - test and run Locust, mockAP
   - run load tests in a kind cluster
- Docs for MockAPI and Locust usage
- Go script to verify load tests have run successfully
- Load tests added to the build pipeline

Possible future improvements:
- Use the mockAPI for part of the integration tests
- Avoid spinning up the kind cluster twice for integration tests and load tests
- Display load test result charts in the PR after completion

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1 +1 @@
-bin/* filter=lfs diff=lfs merge=lfs -text
```
Comment (Azadehkhojandi) [inline .gitattributes]: What was the reason behind deleting this configuration?

Comment (lawrencegripper) [reply .gitattributes]: My understanding of this line is that if enables git large file system on the ./bin directory. This tripped us up at one point. 

As the bin dir is in the ignore file and lfs didn't appear to be needed we removed the line. 

Sorry if this is something that is needed

**Pull request merged**